spec/ParseUser.spec.js

```diff
@@ -1265,6 +1265,36 @@ describe('Parse.User testing', () => {
     done();
   });
 
+  fit('log in with Facebook and save signed up User with verifyUserEmails=true and preventLoginWithUnverifiedEmail=true', async done => {
+    const emailAdapter = {
+      sendPasswordResetEmail: () => Promise.resolve(),
+      sendMail: () => Promise.resolve(),
+    };
+    await reconfigureServer({
+      appName: 'unused',
+      verifyUserEmails: true,
+      preventLoginWithUnverifiedEmail: true,
+      emailAdapter: {
+        module: emailAdapter,
+      },
+      publicServerURL: 'http://localhost:8378/1',
+    });
+
+    const provider = getMockFacebookProvider();
+    Parse.User._registerAuthenticationProvider(provider);
+    const user = await Parse.User._logInWith('facebook');
+    expect(user instanceof Parse.User).toBeTrue();
```

Comment:
This and the following 2 lines are probably also not needed to be checked for this test